tests/test_tra_strings.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
```

Comment:
move this directory inside the mod (/`CompassOfWomanhood`) directory, same with `/utils`

Reply:
As discussed --- `CompassOfWomanhood` is the directory that is copied into the game directory of the user, so I think it's better to avoid placing tools or tests in there. Same as we did with LandingPage.

Unfortunately, it's not just a community convention, but actually a built-in feature of WeiDU .exe files, so going against that would be a bit problematic.